extensions/firmware/org.eclipse.smarthome.firmware.filesystem.test/.settings/org.eclipse.jdt.groovy.core.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
+groovy.compiler.level=-1
```

Comment:
Please remove this file because it causes the default Eclipse project settings to be overridden. See also #6106.